data/migrations/dba_operator.py

```diff
@@ -0,0 +1,142 @@
+import yaml
+import logging
+
+from functools import partial, wraps
+from alembic.operations import ops
+
+import sqlalchemy as sa
+
+
+logger = logging.getLogger(__name__)
+
+
+class Migration(object):
+    def __init__(self):
+        self._schema_hints = []
+
+    def dump_yaml_and_reset(self, stream, revision, down_revision):
+        migration_object = {
+            "apiVersion": "dbaoperator.app-sre.redhat.com/v1alpha1",
+            "kind": "DatabaseMigration",
+            "metadata": {"name": revision,},
+            "spec": {
+                "previous": down_revision,
+                "migrationContainerSpec": {
+                    "name": revision,
+                    "image": "quay.io/quaymigrations/realtest",
```

Comment:
What is `quay.io/quaymigrations/realtest`?

Reply:
I guess I should make that config, and set it to `quay.io/quay/quay` for now? I wonder if it should be config to the migration generator or to the dba operator.